controller-server/src/main/java/com/yahoo/vespa/hosted/controller/ApplicationController.java

```diff
@@ -219,7 +219,7 @@ public Application createApplication(ApplicationId id, Optional<NToken> token) {
 
     /** Deploys an application. If the application does not exist it is created. */
     // TODO: Get rid of the options arg
-    public ActivateResult deployApplication(ApplicationId applicationId, Zone zone,
+    public PrepareResponse deployApplication(ApplicationId applicationId, Zone zone,
```

Comment:
"PrepareResponse" seems wrong as the method does both prepare and active on the config server.